content/docs/atom-guide/look-dev/six-point-lighting-tutorial.md

```diff
@@ -0,0 +1,278 @@
+   1. Specular lighting simulates the bright spot on a shiny object that most brightly reflects light into the camera. In our case, we do not need any specular lighting because 1) we can't effectively apply specular lighting on our 2D textures and 2) our use cases do not require it, as we are mainly using this technique for clouds and non-shiny objects.
+   1. Edit the `GetSpecularLighting` function so we don't have any specular lighting:
+      ```hlsl
+      float3 GetSpecularLighting(Surface surface, LightingData lightingData, const float3 lightIntensity, const float3 dirToLight)
+      {
+         return float3(0.0f, 0.0f, 0.0f);
+      }
+      ```
+1. Great, now let's work on `GetDiffuseLighting`:
+   1. Diffuse lighting simulates how light from an incoming direction scatters. For six-point lighting, we want to use diffuse lighting, since we want to take the light's direction and apply it to figure out our light map.
+   1. Before we edit `GetDiffuseLighting`, let's write a helper function to compute the light map (adapted from [here](https://realtimevfx.com/t/smoke-lighting-and-texture-re-usability-in-skull-bones/5339)):
+      ```hlsl
+      float ComputeLightMap(const float3 dirToLightWS, const Surface surface)
+      {
+         float3 dirToLightTS = WorldSpaceToTangent(dirToLightWS, surface.normal, surface.tangent, surface.bitangent);
+         float hMap = (dirToLightTS.x > 0.0f) ? (surface.right) : (surface.left);   // Picks the correct horizontal side.
+         float vMap = (dirToLightTS.y > 0.0f) ? (surface.bottom) : (surface.top);   // Picks the correct vertical side.
+         float dMap = (dirToLightTS.z > 0.0f) ? (surface.frontside) : (surface.backside);  // Picks the correct front/back side
+         float lightMap = hMap*dirToLightTS.x*dirToLightTS.x + vMap*dirToLightTS.y*dirToLightTS.y + dMap*dirToLightTS.z*dirToLightTS.z; // Pythagoras!
+         return lightMap;
+      }
+      ```
+   1. Now that we have the helper function, we can use `GetDiffuseLighting` to call the function and apply the results:
+      ```hlsl
+      float3 GetDiffuseLighting(Surface surface, LightingData lightingData, float3 lightIntensity, float3 dirToLight)
+      {
+         float lightMap = ComputeLightMap(dirToLight, surface);
+         float3 diffuse = lightMap.rrr;
+         
+         diffuse *= lightIntensity;
+         return diffuse;
+      }
+      ```
+
+Great, now all the custom surface and lighting is now done! In the **Editor**, look at how your material has lighting now. Try adding more entities with a **Directional Light** component around your material to see the different effects. For example, try moving the light to point to the top of your material and see how the lighting responds accordingly! Also, adjust the **Intensity** of the light in the **Directional Light** component as needed to make your cloud look more realistic. Your material will also respond to other light types and multiple lights at the same time.
```

Comment:
"In the Editor, look at how your material has lighting now" -> "Your material should now have lighting in the Editor"